packages/@biomejs/tailwindcss-config-analyzer/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "tailwindcss-config-analyzer",
+  "version": "1.0.0",
+  "description": "Tools to introspect Tailwind CSS configurations for integration with Biome",
+  "private": true,
+	"license": "MIT OR Apache-2.0",
+  "author": "Dani Guardiola",
+  "type": "module",
+  "scripts": {
+    "gen:tailwind-preset": "bun src/generate-tailwind-preset.ts"
+  },
```

Comment:
We use [`just`](https://github.com/biomejs/biome/blob/main/justfile) to launch scripts, we should stick with that.

Reply:
Would be happy to do that, though the reason I'm doing this is because I'm using bun to run the codegen script, so this is like doing `npx bun script.ts` to use the locally installed dep (bun). Not sure how it can be achieved with `just`, could you help me figure it out?

Reply:
`just` is the same as defining a script in the package.json. You give it a name and then run whatever tool you need. 

In this case, it will run bun for the code gen. 

Instead of `npx gen-tailwind` it will be `just gen-tailwind`.

Reply:
Alright, let's do that though it'll require having Bun installed globally, I'll make sure to document this somewhere once we figure out the final location of the codegen script.